client/packages/host/src/components/Login/Login.tsx

```diff
@@ -39,6 +40,7 @@ export const Login = () => {
 
   useEffect(() => {
     setPageTitle(`${t('app.login')} | ${t('app')} `);
+    LocalStorage.removeItem('/auth/error');
```

Comment:
offroad: I had the server error set, and couldn't shake it! With this - reset the auth error when you load the login page